m3u.go

```diff
@@ -33,31 +33,36 @@ type Track struct {
 }
 
 // Parse parses an m3u playlist with the given file name and returns a Playlist
-func Parse(fileName string) (playlist Playlist, err error) {
+func Parse(fileName string) (Playlist, error) {
 	var f io.ReadCloser
-	var data *http.Response
+
 	if strings.HasPrefix(fileName, "http://") || strings.HasPrefix(fileName, "https://") {
-		data, err = http.Get(fileName)
+		data, err := http.Get(fileName)
+		if err != nil {
+			return Playlist{},
```

Comment:
Bit of a nitpick - but shouldn't we return `nil` for the playlist whenever we hit an error condition?

Reply:
Yes, it's better, but the function prototype do not return a pointer...
So if you want to introduce a breaking change, I can do another commit ;)

Reply:
I introduced the breaking change, https://github.com/jamesnetherton/m3u/pull/6/commits/a098a0b466d11947b276c4c80b8b38dbd5757ee5